pulsar-broker/src/main/java/org/apache/pulsar/broker/service/SystemTopicBasedTopicPoliciesService.java

```diff
@@ -480,7 +485,7 @@ boolean checkReaderIsCached(NamespaceName namespaceName) {
     }
 
     @VisibleForTesting
```

Comment:
Since it has been changed to public, can it be deleted here

Reply:
Maybe we can keep it to indicate it's only used by test cases?